generic_e2e_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2012-2020 The GoSNMP Authors. All rights reserved.  Use of this
+// Copyright 2012-present The GoSNMP Authors. All rights reserved.  Use of this
```

Comment:
It's actually fine to [remove the end year completely](https://www.copyrightlaws.com/copyright-symbol-notice-year/#:~:text=The%20general%20rule%20is%20that,first%20publication%20of%20the%20work.) for this kind of notice. Only the initial copyright year is necessary.

```suggestion
// Copyright 2012 The GoSNMP Authors. All rights reserved.  Use of this
```